client/state/data-layer/wpcom/read/tags/index.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * Internal dependencies
+ */
+import {
+	READER_FETCH_TAGS_REQUEST,
+	READER_FETCH_TAG_REQUEST,
+} from 'state/action-types';
+import {
+	receiveTags,
+	receiveTag,
+} from 'state/reader/tags/items/actions';
+
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+
+export function requestTag( store, action, next ) {
+	store.dispatch( http( {
```

Comment:
I plan on holding off on inflight deduping for now.
While I do think its a blocker for `connect`-ing, I don't think it needs to be in this yet.

Theres been a bit of churn and I want to save it for the last step seeing as it could become irrelevant with the deduping in the http layer